pkg/planner/core/planbuilder_test.go

```diff
@@ -924,13 +924,26 @@ func TestBuildAdminAlterDDLJobPlan(t *testing.T) {
 	require.True(t, ok)
 	require.Equal(t, cons.Value.GetString(), "10MiB")
 
-	stmt, err = parser.ParseOneStmt("admin alter ddl jobs 4 thread = 16, batch_size = 512, max_write_speed = '10MiB' ", "", "")
+	stmt, err = parser.ParseOneStmt("admin alter ddl jobs 4 max_write_speed = 1024", "", "")
 	require.NoError(t, err)
 	p, err = builder.Build(ctx, resolve.NewNodeW(stmt))
 	require.NoError(t, err)
 	plan, ok = p.(*AlterDDLJob)
 	require.True(t, ok)
 	require.Equal(t, plan.JobID, int64(4))
+	require.Equal(t, len(plan.Options), 1)
+	require.Equal(t, plan.Options[0].Name, AlterDDLJobMaxWriteSpeed)
```

Comment:
the parameter of `require.Equal` shoud be `t, expected, actual`.

```suggestion
	require.Equal(t,AlterDDLJobMaxWriteSpeed,  plan.Options[0].Name)
```

When the test failed, it will show which value is expected and which value is we got.